.github/workflows/release-core.yml

```diff
@@ -80,28 +68,27 @@ jobs:
           else
             npm publish --tag ${{ github.event.inputs.tags }}
           fi
+        working-directory: packages/core
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: Core - Create PR
-        uses: peter-evans/create-pull-request@v3
-        with:
-          title: release/@scania/tegel@${{ steps.version.outputs.version }}
-          commit-message: Release of @scania/tegel@${{ steps.version.outputs.version }}
-          branch: release/${{ steps.version.outputs.version }}
-          delete-branch: true
+      - name: Core - Create git tag
+        run: git tag @scania/tegel@${{ steps.version.outputs.version }}
+
+      - name: Core - Push git tag
+        run: git push origin @scania/tegel@${{ steps.version.outputs.version }}
+
+
   on-failure:
     needs: release-core
     runs-on: ubuntu-latest
 
     steps:
       - name: Remove branch and git tag on failure
         env:
-          npm_version: ${{ needs.release.outputs.version }}
+          npm_version: ${{ needs.release-core.outputs.version }}
         run: |
           if [ "${{ job.status }}" == "failure" ]; then
```

Comment:
Is this redundant since its already on-failure, or is on-failure just a name?

Reply:
on-failure is just a name